plugins-bundled/internal/plugin-catalog-app/src/components/Card.tsx

```diff
@@ -15,12 +15,12 @@ export const Card = ({ href, text, image, layout = 'vertical' }: Props) => {
   const styles = getCardStyles(theme, layout);
 
   return (
-    <a href={href} className={styles.root}>
+    <Link href={href} className={styles.root}>
```

Comment:
@dprokop -- should using Link make the links update when clicked? or should `<a ` be OK?

Reply:
@ryantxu both should be OK as we intercept `a` link clicks and proxy those to history service.